File: 프로그래머스/1/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    var answer = 0;
    var minxvalue = 0;
    score.sort((a,b)=>b-a);
    for(let i=0; i<score.length; i+=m){
        if(i+m > score.length){
            break;
        }
        minvalue = 0;
        minvalue =Math.min(...score.slice(i,i+m));
        answer += minvalue*m
    }
    return answer;
}

File: 프로그래머스/1/135808. 과일 장수/과일 장수.java
import java.util.*;

class Solution {
    public int solution(int k, int m, int[] score) {
        int answer = 0;
        
        Arrays.sort(score);

        for(int i=0; i<score.length/2; i++){
            int temp = score[i];
            score[i] = score[score.length-1-i];
            score[score.length-1-i] = temp;
        }
        
        for(int i=0; i<score.length; i+=m){
            if(i+m-1>=score.length){
                break;
            }
            
            answer += m*score[i+m-1];
        }

        return answer;
    }
}